task_02/ex4.py

```diff
@@ -0,0 +1,21 @@
+"""Easy Unpack."""
+
+# https://py.checkio.org/en/mission/easy-unpack/
+
+
+def easy_unpack(tuple_arg):
+    """Get an tuple and returns a tuple with 3 elements - first, third.
+
```

Comment:
It's nice to put task conditions to docstring. It would be no need for such a tricks as dividing one sentence into two and dot in strange places. You can put task link to docstring too.